pallets/schemas/src/lib.rs

```diff
@@ -131,5 +146,29 @@ pub mod pallet {
 			);
 			Ok(bounded_fields)
 		}
+
+		pub fn calculate_schema_cost(schema: Vec<u8>) -> FeeDetails<BalanceOf<T>> {
+			let schema_len = schema.len() as u32;
+			let schema_weight = T::WeightInfo::register_schema(schema_len);
+			let unadjusted_weight_fee = T::WeightToFee::calc(&schema_weight);
+			// TODO: for issue: #77
+			//let multiplier = Self::next_fee_multiplier();
+			//let adjusted_weight_fee = multiplier.saturating_mul_int(unadjusted_weight_fee);
+			//let extrinsic_weight = T::BlockWeights::get().get(Self).base_extrinsic;
+			//let base_fee = T::WeightToFee::calc(&extrinsic_weight);
+			// TODO fixed len fee updates if this is relevant for mrc
+			// TODO add multiplier to fee if schema registration will have increasing cost
+			let fixed_len_fee = 0u32.into();
+			let tip = 0u32.into();
+			let base_fee = 0u32.into();
+			FeeDetails {
+				inclusion_fee: Some(InclusionFee {
+					base_fee: base_fee,
+					len_fee: fixed_len_fee,
+					adjusted_weight_fee: unadjusted_weight_fee,
+				}),
+				tip,
+			}
+		}
```

Comment:
This calculation needs to be refined, this is separated in issue #77 , also will be useful as a reference for other pallets needing  such computation